app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,33 @@
+class MenusController < ApplicationController
+  before_action :set_menu, only: [:show, :edit, :update]
+  load_and_authorize_resource
+
+  def index
+    @menus = Menu.all
+  end
+
+  def show
+  end
+
+  def edit
+    redirect_to menus_path unless @menu.for_today?
+  end
+
+  def update
+    if @menu.update(menu_params)
+      redirect_to menus_path and return
+    end
+
+    render :new
+  end
+
+  private
+
+  def set_menu
+    @menu = Menu.find(params[:id])
+  end
+
+  def menu_params
+    params.require(:menu).permit(:menu_type, :name, :price, menu_items_attributes: MenuItem.attribute_names.map(&:to_sym).push(:_destroy))
```

Comment:
`MenuItem.attribute_names.map(&:to_sym)` 
wouldn't recommend this way even when you are going to permit everything. You wouldn't want to permit changing :id for example
Please use explicit list of attributes